db/migrate/20190912143132_add_release_field_to_os.rb

```diff
@@ -0,0 +1,40 @@
+class AddReleaseFieldToOs < ActiveRecord::Migration[5.2]
+  class Operatingsystem < ApplicationRecord
+    self.inheritance_column = nil
+  end
+
+  def up
+    add_column :operatingsystems, :release, :string, :limit => 16, :default => "", :null => false
+
+    ok = true
+    Operatingsystem.all.each do |os|
+      if os.minor&.include?('.')
+        minor_release_array = os.minor.split('.')
+        minor = minor_release_array[0]
+        release = minor_release_array[1]
+        os.update_attribute :minor, minor
+        os.update_attribute :release, release
+        ok &&= os.valid?
```

Comment:
Are there cases, where it can fail? If not, I would use `os.update_columns(minor: minor, release: release)` and ignore the ok all together, if it can, `ok &= os.update(minor: minor, release: release)`. I would prefer former.
Note: I don't believe it can be invalid, when we are using fake model - it has no validations defined.